x-pack/plugins/aiops/public/components/explain_log_rate_spikes/explain_log_rate_spikes_analysis.tsx

```diff
@@ -94,11 +105,34 @@ export const ExplainLogRateSpikesAnalysis: FC<ExplainLogRateSpikesAnalysisProps>
         onRefresh={startHandler}
         onCancel={cancel}
       />
+      <EuiSpacer size="xs" />
+      {errors.length > 0 && (
+        <EuiCallOut
+          title={i18n.translate('xpack.aiops.analysis.errorCallOutTitle', {
+            defaultMessage:
+              'The following {errorCount, plural, one {error} other {errors}} occurred running the analysis.',
+            values: { errorCount: errors.length },
+          })}
+          color="warning"
+          iconType="alert"
+          size="s"
+        >
+          <EuiText size="s">
+            {errors.length === 1 && <p>{errors[0]}</p>}
+            {errors.length > 1 && (
+              <ul>
+                {errors.map((e, i) => (
+                  <li key={i}>{e}</li>
+                ))}
+              </ul>
+            )}
```

Comment:
```suggestion
            {errors.length === 1 ? <p>{errors[0]}</p> : <ul>
                {errors.map((e, i) => (
                  <li key={i}>{e}</li>
                ))}
              </ul>}
```

Reply:
Updated in [b0ab19e](https://github.com/elastic/kibana/pull/137947/commits/b0ab19ed78a883f72a633bf1989e723e04d3c147).